src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,115 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+    @Id
+    private String genre;
+    @Column(nullable = false)
```

Comment:
The @column annotations are required to be able to generate the model in the database.
In this case I would add the @column (nulleable = false) annotation in conjunction with the @NotNull annotation.

The parameter **(nulleable = false)** defines and validates that the field of the table in question is not null. This validation is carried out in persistence, that is, in communication with the database.

@NotNull checks that the field is not null in the java layer, that is, before communicating to the database. This avoids sending invalid data to the database.

```suggestion
    @NotNull 
    @Column(nullable = false)
```
Add this annotation in all the fields that have nulleable = false.

Reply:
I added @NotNull in the mandatory fields